packages/expansion-hub/src/internal/ExpansionHub.ts

```diff
@@ -49,6 +54,18 @@ export class ExpansionHubInternal implements ExpansionHub {
         return this.nativeRevHub.open(serialPort, destAddress);
     }
 
+    get isOpen(): boolean {
+        return this.isOpened();
```

Comment:
It looks like we can delete the `isOpened()` method in `ExpansionHubInternal` and call `this.nativeRevHub.isOpened()` directly from `get isOpen()`.